invesalius/net/pedal_connection.py

```diff
@@ -40,32 +40,32 @@ def __init__(self):
 
         self._midi_in = None
         self._active_inputs = None
-        self._callbacks = {}
+        self._callback_infos = []
 
     def _midi_to_pedal(self, msg):
         # TODO: At this stage, interpret all note_on messages as the pedal being pressed,
         #       and note_off messages as the pedal being released. Later, use the correct
         #       message types and be more stringent about the messages.
         #
         if msg.type == 'note_on':
-            if not self._callbacks:
-                print("Pedal pressed, no callbacks registered")
-            else:
-                Publisher.sendMessage('Pedal state changed', state=True)
-
-                for callback in self._callbacks.values():
-                    callback(True)
+            print("Pedal pressed")
```

Comment:
Is the print strictly necessary assuming that we also have the checkbox in the UI that was working quite well? I would think the print useful for testing, but for releast may be removed.